apps/vr-tests/.storybook/main.js

```diff
@@ -8,6 +10,18 @@ module.exports = {
     // disable react-docgen-typescript (totally not needed here, slows things down a lot)
     reactDocgen: false,
   },
-  webpackFinal: config => custom(config),
+  webpackFinal: config => {
+    const tsPaths = new TsconfigPathsPlugin({
+      configFile: path.resolve(__dirname, '../../../tsconfig.base.json'),
+    });
+
+    if (config.resolve) {
```

Comment:
we should call the `custom` first then set the plugins to that modified behaviour

Reply:
From what I've tested, it didn't seem to affect the default as the plugin doesn't seem to change it at that point in time (i.e. aliases are not affected).
Went with this to have a simplified logic but can revert back to the original solution we had. Wdyt?

Reply:
yeah as I said, not a biggie. you have green light ;)